ethereum-consensus/src/deneb/blob_sidecar.rs

```diff
@@ -27,6 +27,28 @@ pub struct BlobSidecar<
     pub kzg_commitment_inclusion_proof: Vector<Bytes32, KZG_COMMITMENT_INCLUSION_PROOF_DEPTH>,
 }
 
+impl<const BYTES_PER_BLOB: usize, const KZG_COMMITMENT_INCLUSION_PROOF_DEPTH: usize>
+    BlobSidecar<BYTES_PER_BLOB, KZG_COMMITMENT_INCLUSION_PROOF_DEPTH>
+{
+    pub fn verify_blob_sidecar_inclusion_proof(&mut self) -> bool {
+        // TODO: Calculate real gindex
+        let gindex = 4;
```

Comment:
lets just define index once, this may change a bit after I land the ssz-rs updates
```suggestion
        let index = 4;
```